plugin/commentary.vim

```diff
@@ -44,6 +44,12 @@ function! s:go(...) abort
     endif
   endfor
 
+  if exists('b:commentary_startofline') && b:commentary_startofline == 1
+    let rx_indent = '^'
```

Comment:
What does `rx` stand for?

Reply:
`rx` is for regexp. I can remove it or change to whatever you prefer

Reply:
Let's just go with `indent`.

Reply:
Ok